src/core/markdown/__test__/generating-custom-object-docs.spec.ts

```diff
@@ -46,7 +47,6 @@ describe('Generates Custom Object documentation', () => {
       expect(result).documentationBundleHasLength(1);
       assertEither(result, (data) => expect(data).firstDocContains('`TestObject__c`'));
     });
-
```

Comment:
[nitpick] Let's revert this. As a convention we are adding a blank line of separation between blocks